src/web/components/Card/Card.tsx

```diff
@@ -146,9 +147,21 @@ export const Card = ({
                                         pillar={pillar}
                                         size={headlineSize}
                                         showQuotes={showQuotes}
-                                        kickerText={kickerText}
-                                        showPulsingDot={showPulsingDot}
-                                        showSlash={showSlash}
+                                        kickerText={
+                                            designType === 'Live'
+                                                ? 'Live'
+                                                : kickerText
+                                        }
+                                        showPulsingDot={
+                                            designType === 'Live' && isLiveBlog
```

Comment:
Does designType `live` not imply `isLiveBlog` always true?

Reply:
I thought you could have designType = Live but not isLiveBlog if you were a live blog but are no longer active but now I'm not so sure.

https://www.theguardian.com/us-news/live/2020/feb/03/democratic-race-starts-in-earnest-with-iowa-caucuses-live-coverage used to be a live blog but now has `designType = Article` and `isLiveBlog = true`

Reply:
Yeah, designType on `deadBlogs` is `Article` and you'd only show the pulsing dot and `live` kicker on a live liveblog?

Reply:
So, my logic was off - thanks for highlighting this! It's just designType that matters here so I've made a change to reflect this

Reply:
Yes, dead blogs are treated like plain Articles (at least in cards anyway). They just have 'as it happened' at the end of the headline

https://www.theguardian.com/us-news/2020/feb/03/americans-shut-out-iowa-caucuses